src/api/app/components/sourcediff_component.rb

```diff
@@ -0,0 +1,22 @@
+class SourcediffComponent < ApplicationComponent
+  attr_accessor :bs_request, :action, :index, :refresh
+
+  delegate :diff_label, to: :helpers
+  delegate :diff_data, to: :helpers
```

Comment:
Huh, didn't know about this! I just `include WhateverHelperINeed`

Reply:
Including isn't needed, helpers are always available behind `helpers.my_helper_123`. This is documented in our [developer wiki](https://github.com/openSUSE/open-build-service/wiki/View-Components#how-to-use-custom-helpers-in-view-components).